src/AzureSqlAction.ts

```diff
@@ -127,8 +127,9 @@ export default class AzureSqlAction {
             case SqlPackageAction.Publish: 
             case SqlPackageAction.Script:
             case SqlPackageAction.DeployReport:
+                args += `/SourceFile:"${inputs.filePath}"`
```

Comment:
The `/Action` and `/TargetConnectionString` are missing here

Reply:
Taken care of, thank you.